### ISSUE TITLE
Steering wheel controller is not recognised as a device

### ISSUE THREAD

**Issue by MintArcade** [godotengine/godot#31200]
**Godot version:3.1.1**

**OS/device including version: Windows 10**

**Input device in question:  Logitech Driving Force EX**

**Issue description:**
Steering wheel controller is not recognised by game engine
![Annotation 2019-08-05 214946](https://user-images.githubusercontent.com/10226177/62686858-78505980-b9bd-11e9-8cae-3f15acbb1e40.png)

**Steps to reproduce:**
1. Download and install Joypads Demo / Tool from Templates
2. Connect Steering wheel controller (if not connected already)
3. Press play. Steering wheel controller will not appear in the list of available devices.

**Comment by KoBeWi** [comment]
Can anyone still reproduce this bug in Godot 3.2.3 or any later release?

**Comment by Sslaxx** [comment]
@KoBeWi - check the pull @BastiaanOlij has made - not sure if that's compatible with this model of controller though.

**Comment by BastiaanOlij** [comment]
Yes in theory #44082 should fix this.

**Comment by MintArcade** [comment]
Yes. Different computer with no drivers - Godot recognized wheel as a joystick. After installing driver Godot stopped recognizing it at all. Drivers you can separate pedals into two different axis (burnout) and rumble. Will try again in in 3.2.f after #44082 merge. Thank you very much.

**Comment by Sslaxx** [comment]
Can anyone confirm if this is fixed by #44082 in latest master builds?
